example_configs/dropwizard-0-8-x.yml

```diff
@@ -0,0 +1,153 @@
+---
+lowercaseOutputLabelNames: true
+lowercaseOutputName: true
+blackListObjectNames: ["metrics:type=*,name=jvm*"]
+
+rules:
+
+- pattern: "metrics<name=ch.qos.logback.core.Appender.(debug|error|info|trace|warn)><>Count"
+  name: log_msg_total
+  labels:
+    level: "$1"
+  type: COUNTER
+
+- pattern: "metrics<name=ch.qos.logback.core.Appender.(debug|error|info|trace|warn)><>(FiveMinuteRate|MeanRate|OneMinuteRate|FifteenMinuteRate)"
+  name: log_msg_rate
+  labels:
+    level: "$1"
+    calculation: "$2"
+  type: COUNTER
+
+- pattern: "metrics<name=ch.qos.logback.core.Appender.all><>Count"
+  name: log_msg_all_total
+  type: COUNTER
+
+- pattern: "metrics<name=ch.qos.logback.core.Appender.all><>(FiveMinuteRate|MeanRate|OneMinuteRate|FifteenMinuteRate)"
+  name: log_msg_all_rate
+  labels:
+    calculation: "$1"
+  type: COUNTER
+
+- pattern: "metrics<name=org.eclipse.jetty.server.HttpConnectionFactory.(.+).connections><>Count"
+  name: http_connections_total
+  type: COUNTER
+  labels:
+    port: "$1"
+
+
+- pattern: "metrics<name=io.dropwizard.jetty.MutableServletContextHandler.(.+)-responses><>Count"
+  name: http_response_codes_total
+  type: COUNTER
+  labels:
+    http_code: "$1"
+
+- pattern: "metrics<name=io.dropwizard.jetty.MutableServletContextHandler.active-(.+)><>Count"
```

Comment:
The first one lists count of opened http connections on different ports.
The middle-one lists ´1xx|2xx|4xx|5xx`